doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1153,6 +1153,37 @@ reduction will only be performed when the argument in position of the
 decreasing argument (which type should be in an inductive definition)
 starts with a constructor.
 
+The check of decreasing argument can be temporary disabled using ``Unguarded``.
+Warning: this can break the consistency of the system, use at your own risk.
```

Comment:
This documentation is not done in the recommended style, the documentation must come after the variant grammar and must be indented. Since this chapter is heavily reindented by #7537 anyways, I suggest you rebase on top or wait for it to get merged. In any case, you will have so many conflicts in this file that the best way to solve them will be to copy / paste this part.

Reply:
Ok thanks, I will do this after merge then.